test/conformance/device/urDeviceGetGlobalTimestamps.cpp

```diff
@@ -102,7 +103,12 @@ TEST_F(urDeviceGetGlobalTimestampTest, SuccessSynchronizedTime) {
         const uint64_t allowedDiff = static_cast<uint64_t>(
             std::min(deviceTimeDiff, hostTimeDiff) * allowedTimerError);
 
-        ASSERT_LE(observedDiff, allowedDiff);
```

Comment:
Why not change the `allowdDiff`/`allowedTimerError`?

Reply:
The problem of changing allowDiff is that there is no amount of time that is guaranteed to work.

I did run the test locally many times and there is some pretty big outliers. I suspect that it might be due to context switches in the OS which can take an unpredictable amount of time.

So I could change allowDiff to some huge number and the test would still not be guaranteed to pass 100% of the time.